README.md

```diff
@@ -1 +1,140 @@
-# AsylumAdvice app
+<!-- PROJECT SHIELDS --> <!-- *** I'm using markdown "reference style" links for readability. *** Reference links are enclosed in brackets [ ] instead of parentheses ( ). *** See the bottom of this document for the declaration of the reference variables *** for contributors-url, forks-url, etc. This is an optional, concise syntax you may use. *** https://www.markdownguide.org/basic-syntax/#reference-style-links -->
+
+[![Contributors][contributors-shield]][contributors-url] [![Forks][forks-shield]][forks-url] [![Stargazers][stars-shield]][stars-url] [![Issues][issues-shield]][issues-url] [![MIT License][license-shield]][license-url]
+
+<!-- PROJECT LOGO --><br>
+
+[![Logo](public/images/asylum-advice-logo.svg)](https://github.com/AsylumAdvice/app)
+
+# Asylum Advice
+
+Asylum Advice helps asylum seekers find the information that is relevant to their asylum procedure.
```

Comment:
nit pick: maybe "Asylum Advice helps asylum seekers find the information that is relevant to their asylum procedure: finding advisors and lawyers" would be better and the 2nd sentence could be removed?